public/data/aryasoni.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "Arya Soni",
+  "bio": "I learn to satisfy my thoughts. I am always looking for opportunities to meet new people and work together. I like solving problems and creating solutions.",
+  "avatar": "https://avatars.githubusercontent.com/u/18515597?s=400&u=4ecfe979f461f79e86a9c7a26183c1327da5d2bc&v=4",
+  "links": [
+    {
+      "name": "Follow me on GitHub",
+      "url": "http://github.com/aryasoni98",
+      "icon": "github"
+    },
+    {
+      "name": "Follow me on Twitter",
+      "url": "http://twitter.com/aryasoni98",
+      "icon": "twitter"
+    }
+  ]
+}
```

Comment:
This should be in a separate PR to other app changes as there are unrelated and both add different value

Reply:
Okay 👍 I will make it